sormas-api/src/main/java/de/symeda/sormas/api/utils/DateHelper.java

```diff
@@ -1004,4 +1004,48 @@ public static Timestamp toTimestampUpper(Date date) {
 		timestamp.setNanos(timestamp.getNanos() + 999999);
 		return timestamp;
 	}
+
+	/**
+	 * Anzahl Millisekunden pro Sekunde.
```

Comment:
Translate to English; Also I'm not sure whether "Millies" is a common abbreviation in English, I'd prefer to use "Milliseconds" everywhere

Reply:
fixed